package.json

```diff
@@ -72,5 +71,8 @@
     "react-html-parser": "^2.0.2",
     "react-stickynode": "^2.1.1",
     "rehype-sectionize-headings": "^1.0.0-rc.1"
+  },
+  "optionalDependencies": {
+    "@mapbox/mapbox-assembly": "^0.26.0"
```

Comment:
Oh! I think we can update to `"@mapbox/mbx-assembly": "^0.28.0"` (or higher), which is when we switched the mapbox assembly package to public. https://github.com/mapbox/mbx-assembly/blob/master/mapbox-changelog.md#0280

Reply:
🤦‍♂ thank u colleen